diff --git a/backend/src/backend/gen/gen_mesa_disasm.c b/backend/src/backend/gen/gen_mesa_disasm.c
index 5653275..fb7deb1 100644
--- a/backend/src/backend/gen/gen_mesa_disasm.c
+++ b/backend/src/backend/gen/gen_mesa_disasm.c
@@ -47,6 +47,7 @@
 #include <stdarg.h>
 #include <stdint.h>
 #include <assert.h>
+#include <sys/types.h>
 
 #include "backend/gen_defs.hpp"
 #include "backend/gen7_instruction.hpp"
diff --git a/src/cl_alloc.c b/src/cl_alloc.c
index e532569..cd622b3 100644
--- a/src/cl_alloc.c
+++ b/src/cl_alloc.c
@@ -23,6 +23,7 @@
 #include <stdlib.h>
 #include <assert.h>
 #include <malloc.h>
+#include <inttypes.h>
 
 static volatile int32_t cl_alloc_n = 0;
 
diff --git a/src/cl_api.c b/src/cl_api.c
index d0d4dc5..36c6c4d 100644
--- a/src/cl_api.c
+++ b/src/cl_api.c
@@ -41,6 +41,7 @@
 #include <string.h>
 #include <assert.h>
 #include <unistd.h>
+#include <sys/types.h>
 
 #include "performance.h"
