diff -Naur 0ad-0.0.20-alpha.old/source/third_party/cppformat/format.cpp 0ad-0.0.20-alpha/source/third_party/cppformat/format.cpp
--- 0ad-0.0.20-alpha.old/source/third_party/cppformat/format.cpp	2016-08-08 20:03:22.570725603 +0200
+++ 0ad-0.0.20-alpha/source/third_party/cppformat/format.cpp	2016-08-08 19:58:46.709733449 +0200
@@ -431,7 +431,7 @@
     int error_code, char *&buffer, std::size_t buffer_size) FMT_NOEXCEPT(true) {
   assert(buffer != 0 && buffer_size != 0);
   int result = 0;
-#if defined(_GNU_SOURCE) && !defined(__BIONIC__)
+#if defined(__GLIB__)
   char *message = strerror_r(error_code, buffer, buffer_size);
   // If the buffer is full then the message is probably truncated.
   if (message == buffer && strlen(buffer) == buffer_size - 1)
diff -Naur 0ad-0.0.20-alpha.old/source/lib/sysdep/os/linux/ldbg.cpp 0ad-0.0.20-alpha/source/lib/sysdep/os/linux/ldbg.cpp
--- 0ad-0.0.20-alpha.old/source/lib/sysdep/os/linux/ldbg.cpp	2016-08-08 20:36:36.796668890 +0200
+++ 0ad-0.0.20-alpha/source/lib/sysdep/os/linux/ldbg.cpp	2016-08-08 20:37:11.142667913 +0200
@@ -33,7 +33,7 @@
 #include "lib/sysdep/sysdep.h"
 #include "lib/debug.h"
 
-#if OS_ANDROID
+#ifndef __GLIB__
 
 // Android NDK doesn't support backtrace()
 // TODO: use unwind.h or similar?
