From 3712532a79721dc28cd98a18d9793c52cf52a181 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Van=C4=9Bk?= <vanekpe5@fel.cvut.cz>
Date: Wed, 8 May 2013 20:47:10 +0200
Subject: [PATCH] Added Idsn inertia string definition

---
 v_rep/sourceCode/strings/v_repStringTable.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/v_rep/sourceCode/strings/v_repStringTable.h b/v_rep/sourceCode/strings/v_repStringTable.h
index 08890c0..b034a6d 100644
--- a/v_rep/sourceCode/strings/v_repStringTable.h
+++ b/v_rep/sourceCode/strings/v_repStringTable.h
@@ -45,6 +45,9 @@
 #define IDSN_MAP_COORD_SPHERE "sphere"
 #define IDSN_MAP_COORD_CUBE "cube"
 
+#define IDSN_INERTIA_FRAME "Inertia frame"
+#define IDSN_INERTIA_FRAME_RELATIVE_TO_WORLD "Inertia frame relative to world"
+
 #define IDSN_EXTERNAL_EDITOR_INFO "Make sure to save the settings and to restart V-REP, for changes to take effect!"
 #define IDSN_EXTERNAL_SCRIPT_EDITOR "External script editor"
 
-- 
1.8.1.5

