From b4c71f04a17c0096f08e1c2cefa0814b5086eb99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Van=C4=9Bk?= <vanekpe5@fel.cvut.cz>
Date: Thu, 9 May 2013 09:55:13 +0200
Subject: [PATCH] Corection

---
 v_rep/v_rep.pro | 44 +++++++++-----------------------------------
 1 file changed, 9 insertions(+), 35 deletions(-)

diff --git a/v_rep/v_rep.pro b/v_rep/v_rep.pro
index b96c29e..c47d574 100644
--- a/v_rep/v_rep.pro
+++ b/v_rep/v_rep.pro
@@ -45,41 +45,15 @@ RESOURCES += variousImageFiles.qrc
 
 RESOURCES += imageFiles_vrep.qrc
 
-win32 {
-	INCLUDEPATH += **your boost folder**
-	INCLUDEPATH += **your QScintilla "Qt4" folder**
-	INCLUDEPATH += **your QScintilla "include" folder**
-	INCLUDEPATH += **your lua5.1 "include" folder**
-	LIBS += **your "lua5.1.lib" location**
-	LIBS += **your "libqscintilla2.a" location**
-	DEFINES += WIN_VREP
-}
-
-macx {
-	INCLUDEPATH += **your boost folder**
-	INCLUDEPATH += **your QScintilla "Qt4" folder**
-	INCLUDEPATH += **your QScintilla "include" folder**
-	INCLUDEPATH += **your lua5.1 "include" folder**
-	LIBS += **your "liblua5.1.dylib" location**
-	LIBS += **your "libqscintilla2.dylib" location**
-	DEFINES += MAC_VREP
-	DEFINES += QT_VREP_64BIT
-}
-
 unix:!macx {
-	INCLUDEPATH += **your boost folder**
-	INCLUDEPATH += **your QScintilla "Qt4" folder**
-	INCLUDEPATH += **your QScintilla "include" folder**
-	LIBS += **your "libqscintilla2.so" location**
+	INCLUDEPATH += /usr/include/boost/
+	INCLUDEPATH += /usr/include/qt4/
+	INCLUDEPATH += /usr/include/qt4/Qsci/
+	LIBS += -L/usr/lib/qt4/ -lqscintilla2
 	contains(QMAKE_HOST.arch, x86_64):{ # 64 Bit
-		INCLUDEPATH += **your lua5.1 64"include" folder**
-		LIBS += -Llua5_1_4_Linux26g4_64_lib/ -llua5.1
 		DEFINES += QT_VREP_64BIT
 	}
-	!contains(QMAKE_HOST.arch, x86_64):{ # 32 Bit
-		INCLUDEPATH += **your lua5.1 32"include" folder**
-		LIBS += -Llua5_1_4_Linux26g4_lib/ -llua5.1
-	}
+	LIBS += -llua
 	DEFINES += LIN_VREP
 }
 
@@ -560,7 +534,7 @@ HEADERS += sourceCode/textures/TextureObject.h \
 	sourceCode/textures/TextureProperty.h \
 
 HEADERS += sourceCode/communication/serialPort/SerialPort.h \
-	sourceCode/communication/serial_mac_linux.h \
+	sourceCode/communication/serialPort/serial_mac_linux.h \
 
 HEADERS += sourceCode/serialization/Ser.h \
 	sourceCode/serialization/extIkSer.h \
@@ -595,7 +569,7 @@ HEADERS += sourceCode/menusAndSimilar/ToolBarCommand.h \
 
 HEADERS += sourceCode/geometricAlgorithms/LinMotionRoutines.h \
 	sourceCode/geometricAlgorithms/EdgeCont.h \
-	sourceCode/geometricAlgorithms/algos/MeshRoutines.h \
+	sourceCode/geometricAlgorithms/MeshRoutines.h \
 	sourceCode/geometricAlgorithms/MeshManip.h \
 	sourceCode/geometricAlgorithms/EdgeElement.h \
 	sourceCode/geometricAlgorithms/algos.h \
@@ -624,7 +598,7 @@ HEADERS += sourceCode/geometricAlgorithms/qHull/user.h \
 	sourceCode/geometricAlgorithms/qHull/io.h \
 	sourceCode/geometricAlgorithms/qHull/geom.h \
 
-HEADERS += sourceCode/various/vrepConfig \
+HEADERS += sourceCode/various/vrepConfig.h \
 	sourceCode/various/GV.H \
 	sourceCode/various/Global.h \
 	sourceCode/various/embeddedFonts.h \
@@ -637,7 +611,7 @@ HEADERS += sourceCode/various/vrepConfig \
 	sourceCode/various/DirectoryPaths.h \
 	sourceCode/various/ConstraintSolverObject.h \
 
-HEADERS += sourceCode/libsAndPlugins/pluginContainer.h \
+HEADERS += sourceCode/libsAndPlugins/PluginContainer.h \
 	sourceCode/libsAndPlugins/auxLib.h \
 	sourceCode/libsAndPlugins/auxLibVideo.h \
 
-- 
1.8.1.5

